extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -343,7 +357,9 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
               {activeProductData.aggregateRating.ratingValue} (
               <a
                 class="i-amphtml-amp-story-shopping-pdp-reviews-link"
-                href={activeProductData.aggregateRating.reviewUrl}
+                href={this.resolveRelativeUrl_(
+                  activeProductData.aggregateRating.reviewUrl
+                )}
```

Comment:
As an example for the rest:
```suggestion
                href={resolveRelativeToSourceUrl(
                  this.element,
                  activeProductData.aggregateRating.reviewUrl
                )}
```